tools/install_dependencies.sh

```diff
@@ -11,12 +11,13 @@ if [[ $(lsb_release -rs) != "18.04" ]]; then
   fi
 fi
 
+sudo apt install -y apt-transport-https ca-certificates gnupg software-properties-common wget
```

Comment:
Why do we need the first 4 packages? Put in some comments?

Reply:
Seems they are used for docker setup. Can they be moved to tools/docker/README.md?

Reply:
gnupg, apt-transport, https, ca-certificates are needed to wget over https and then execute `gpg --dearmor` on line 15.